docs/walkthrough.md

```diff
@@ -0,0 +1,551 @@
+But the CMS is in fact rendering this view, and is making an educated guess as to what template it should use. It's falling back to your `base.html` at the moment, but that's not its first choice. Let's create a template called `content/mycontent.html`:
+
+```
+{% extends 'base.html' %}
+
+{% block main %} {# or whatever your main block is on your site :) #}
+  <h1>{{ pages.current.title }}</h1>
+
+  <p>{{ pages.current.content.introduction }}</p>
+{% endblock %}
+```
+
+Now reload your page. It has content! That's because if it hasn't been told to do anything else, it will look for a template at `<app_label>/<model_name>.html`. Convention over configuration ahoy!
+
+Where did `pages` come from? That would be the template context processor you added earlier.
+`pages.current` refers to the currently active Page.
+`pages.current.content` refers to the instance of your content model that is attached to the page.
+
+This is the very simplest example of rendering a page's content model on the front end.
+There's all sorts of other things we can do with content models, but if your model has some fields, and maybe some inlines, you don't _need_ to write any views, just a template.
+
+Did we (royal we) say inlines? We definitely did.
+
+## Lets add some admin inlines
+
+OK, so let's say you want to build your new content page out of an entirely arbitrary number of sections.
+That's fine too!
+We can add inlines to our change-page view in the admin.
+Here is what your model looks like.
+Pay special attention to the ForeignKey if nothing else - this is essential, and note that it is to `pages.Page` and _not_ your content model:
+
+```python
+class ContentSection(models.Model):
+    page = models.ForeignKey(
+        'pages.Page',
+        on_delete=models.CASCADE,
+    )
+
+    title = models.CharField(
+        max_length=100,
+    )
+
+    text = models.TextField(
+        null=True,
+        blank=True,
+    )
+
+    order = models.PositiveIntegerField(
+        default=0,
+    )
+
+    class Meta:
+        ordering = ['order']
+```
+
+We've defined a section model with a title, text, and an ordering field.
+Now let's register it as an inline for MyContent:
+
+```python
+from cms.apps.pages.admin import page_admin
+from django.contrib.admin import StackedInline
+
+from .models import MyContent, ContentSection
+
+
+class ContentSectionInline(StackedInline):
+    model = ContentSection
+
+page_admin.register_content_inline(MyContent, ContentSectionInline)
+```
+
+That's it! It's just as easy as adding inlines to any other model.
+We've told the CMS "be prepared to display these inlines only when the page type is MyContent".
+This won't appear on pages whose type is any other model, because it might not make sense there.
+
+Now, stick this just before the `{% endblock %}` in your `content/mycontent.html` template:
```

Comment:
As much as it's kinda obvious it could be worth mentioning what you should see on the front-end: (all the sections added to the homepage rendered in order)